packages/front/src/routes/association/[identifier]/components/Establishments/Establishments.svelte

```diff
@@ -13,7 +13,9 @@
 
 <div>
     {#if $establishmentsStore.length && $associationStore}
-        <h2>{$establishmentsStore.length} Les établissements rattachés à cette association</h2>
+        <h2>
+            {$establishmentsStore.length} établissement{$establishmentsStore.length > 1 ? "s" : ""} rattachés à cette association
```

Comment:
```suggestion
            {$establishmentsStore.length} {$establishmentsStore.length > 1 ? "établissements rattachés" : "établissement rattaché"}  à cette association
```
Presque, mais il faut accorder l'adjectif aussi. À voir si ça vaut de le faire rentrer dans le contrôleur ou pas